src/Illuminate/Database/Query/JoinClause.php

```diff
@@ -46,8 +46,15 @@ public function __construct($type, $table)
     }
 
     /**
-     * Add an "on" clause to the join.
-     *
+     * Add an "on" clause to the join. On clauses can be chained, e.g.
```

Comment:
could you move "On clauses can be chained, e.g." to the "Long description" please